modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/DeployVariable.java

```diff
@@ -19,7 +20,7 @@
  * Defines for the deploy variable.
  */
 @Data
-public class DeployVariable {
+public class DeployVariable implements Serializable {
 
```

Comment:
Add serialVersionUID to avoid potential incompatibilities during deserialization.